templates/fine-tune-llm/training_configs/full_param/llama-2-70b-4096-4xg5_48xlarge.yaml

```diff
@@ -0,0 +1,18 @@
+model_id: meta-llama/Llama-2-70b-chat-hf # <-- change this to the model you want to fine-tune
+train_path: s3://air-example-data/gsm8k/train.jsonl # <-- change this to the path to your training data
+valid_path: s3://air-example-data/gsm8k/test.jsonl # <-- change this to the path to your validation data. This is optional
+context_length: 4096 # <-- change this to the context length you want to use
+num_devices: 32 # <-- change this to total number of GPUs that you want to use
+num_epochs: 10 # <-- change this to the number of epochs that you want to train for
+train_batch_size_per_device: 8
```

Comment:
I think bsize is too large for this?

Reply:
it has worked for the jobs that i submitted so I am keeping it 8 for now